infogami/plugins/links/code.py

```diff
@@ -18,7 +18,6 @@ class backlinks (delegate.mode):
     def GET(self, site, path):
         #@@ fix later
         return []
-
-        links = tdb.Things(type=db.get_type(site, 'type/page'), parent=site, links=path)
+        # unreachable code...
```

Comment:
This seemed very strange, but the failure to load OL seems to be from this comment line! When I try to import this file in Python 2 I get:

```
>>> import code
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "code.py", line 21
SyntaxError: Non-ASCII character '\xc2' in file code.py on line 21, but no encoding declared; see http://python.org/dev/peps/pep-0263/ for details
```

Deleting that comment fixes the problem. I couldn't see the problem using github, editors, or other tools, and copying the text seemed to ignore the strange characters, but opening this file in a hex editor shows `\xC2 \xA0` bytes between the `#` and `unreachable`, which is a Unicode no-break space in utf8.

The other way to fix would be to declare a utf8 coding in this file fot Python 2, but I'm not sure the space was deliberate?

Annoyingly this load error was _not_ picked up in Travis tests, but then again this file seems to do nothing very useful, but it was loaded by Open Library calling `infogami._setup()` from https://github.com/internetarchive/openlibrary/blob/0565e81e4e2e1ecf2fdc497e9d9bc22bf69b5685/scripts/openlibrary-server#L86

Reply:
Unbalanced parentheses??

Reply:
Oh... There is a space char that is not quite a space char.  If you retype each space char it will go away.

Reply:
I have replaced that space character, and the site loads.

I'm only testing the changes indirectly via OL, but I was able to add a book record with Unicode title, publisher, author and other fields and search and retrieve it in local dev, so no smoke! :)